### ISSUE TITLE
Downloading graphical interface files error

### ISSUE THREAD

**Issue by papernoise** [mapedorr/popochiu#82]
**Godot version**: 4.1.1
**Popochiu Version**: 2.0 alpha
**OS**: Windows 11

## What I did
- Create empty Popochiu project (i.e. add the addon folder, activate plugin, reload project)
- Launch setup, since I want to work at FullHD res, I set everything up for 1920x1080
- create a new room (set it as the initial one)
- add a background prop, so there's something in there

## What happens
All the graphic interface files seem to have been copied to the FileSystem
after clicking the "Move graphic interface" button I get this error messages in the Output:

```
res://popochiu/graphic_interface/dialog_text/dialog_text.gd:93 - Parse Error: The variable type is being inferred from a Variant value, so it will be typed as Variant. (Warning treated as error.)
res://popochiu/graphic_interface/toolbar/btn_quit.gd:0 - Parse Error: Could not resolve super class path "res://addons/popochiu/engine/objects/graphic_interface/toolbar/toolbar_button.gd".
  res://addons/popochiu/popochiu_plugin.gd:231 - Cannot call method 'get_file_count' on a null value.
```
then when I try to run the game, I get this error:

```
Parser Error: The variable type is being inferred from a Variant value, so it will be typed as Variant. (Warning treated as error.)
```
It highlights line 93 in the dialog_text.gd file giving this messages:
`**error**: Line 93:The variable type is being inferred from a Variant value, so it will be typed as Variant. (Warning treated as error.)`
`**Warning**: [Ignore]Line 70 (UNUSED_VARIABLE):The local variable "characters_count" is declared but never used in the block. If this is intended, prefix it with an underscore: "_characters_count".`

Stack trace shows this:

![image](https://github.com/mapedorr/popochiu/assets/1595523/30c2c259-27df-4504-b836-01b9ae1c6fe6)

**Comment by papernoise** [comment]
Btw. the same happens when using the settings from the tutorial, i.e. 320x180px resolution and the "pixel" setting.

**Comment by mapedorr** [comment]
Fixed in 52aac5d0eeea4de048dfdb06cc38dd6dde38b88d
